src/GlobalStyles.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap');
+*box {
```

Comment:
What is this selector?
Maybe you tried this:
```suggestion
*, *:after, *:before {
```